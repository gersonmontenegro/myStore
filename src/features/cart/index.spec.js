/* eslint-disable react/jsx-props-no-spreading */
import React from 'react';
import { cleanup, render } from 'react-native-testing-library';
import Cart from './index';

const props = {
    navigation: {
        navigate: jest.fn(),
    },
};

describe('Test cart interface', () => {
    afterEach(() => cleanup);
    let wrapper;
    beforeEach(() => {
        wrapper = render(<Cart {...props} />);
    });

    test('should render properly', () => {
        expect(wrapper).toMatchSnapshot();
    });

    test('should show subtotal', () => {
        const { queryByText } = wrapper;
        const totalAmount = useSelector(getTotalFromCart);
        expect(queryByText(`Subtotal: ${Format.currencyFormat(totalAmount)}`));
    });

    test('should render # elements', () => {
});
