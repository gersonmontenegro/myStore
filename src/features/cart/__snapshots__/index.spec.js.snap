// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test cart interface should render properly 1`] = `undefined`;
